guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

```diff
@@ -65,7 +65,8 @@ Entitlement Type:    Physical
 . Make a note of the subscription Pool ID.
 Your subscription Pool ID is different from the example provided.
 
-. Attach the {Project} Infrastructure subscription to the base operating system that your {ProductName} is running on:
+. Attach the {Project} Infrastructure subscription to the base operating system that your {ProductName} is running on.
+This step can be skipped if SCA is enabled for {Project}.
```

Comment:
```suggestion
If SCA is enabled for {Project}, you can skip this step:
```

Should this be "on {ProjectServer}" instead of "for Project"?

Reply:
Agree. Addressed.